internal/integration/knot/network/subscriber.go

```diff
@@ -24,17 +24,34 @@ func NewMsgSubscriber(amqp *AMQP) Subscriber {
 
 func (ms *msgSubscriber) SubscribeToKNoTMessages(msgChan chan InMsg) error {
```

Comment:
This is a exported function. This function should have comment.

Reply:
Ok.